addon/inspect.js

```diff
@@ -371,6 +376,16 @@ class Model {
     );
 
   }
+  updateShowTableBorder() {
+    this.showTableBorder = !this.showTableBorder;
+    localStorage.setItem("displayInspectTableBorders", "true"); // Save to local storage
```

Comment:
Hi @pietromartino I think this is what you wanted to do right ? Because actually the preference is not saved
```suggestion
    localStorage.setItem("displayInspectTableBorders", this.showTableBorder); // Save to local storage
```